Magento2/ruleset.xml

```diff
@@ -493,13 +493,6 @@
         <severity>5</severity>
         <type>warning</type>
     </rule>
-    <rule ref="Squiz.Commenting.ClassComment">
```

Comment:
Do we have our custom sniff enabled?

Reply:
Yes, those Annotation sniffs are still under `dev/tests/static`.
Once new will be implemented here, I'll remove the old ones.
https://github.com/magento/magento2/blob/2.3-develop/dev/tests/static/framework/Magento/ruleset.xml#L12